packages/uilib/src/lib/plugins/communication-explorer/sidebar/sidebar.svelte

```diff
@@ -77,18 +95,31 @@
                     I don't think it adds much user value
                     and it is going to be hard to support
                 -->
+                <input
```

Comment:
Shouldn't we use the components from the svelte MUI library to keep consistency in our design ?

Reply:
Just talked with @michelguerin: Implementing MUI is a good idea but needs to be consistent in the sidebar. This can result in major changes that far exceed the scope of this ticket. A changeover to MUI should therefore be planned and implemented separately.